aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -81,6 +81,7 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.type_info.type_name.base, "type_info.type_name.base", 300 * MUL],
     // TODO(Gas): the on-chain name is wrong...
     [.type_info.type_name.per_byte_in_str, "type_info.type_name.per_abstract_memory_unit", 5 * MUL],
+    [.type_info.chain_id.base, "type_info.chain_id.base", 150 * MUL],
```

Comment:
You need to mark this optional to maintain backward compatibility. Please also bump the gas feature version

Reply:
This was the `LATEST_GAS_FEATURE_VERSION` constant in `aptos-move/aptos-gas/src/gas_meter.rs`, right?

Reply:
(If so, then done.)

Reply:
Yes

Reply:
Ugh. One second. I have no idea how, but I seem to have accidentally-reverted my changes...

Reply:
Okay, fixed...